packages/web-components/src/components/ic-link/ic-link.tsx

```diff
@@ -171,7 +192,7 @@ export class Link {
             class={{
               ["link"]: !!href,
             }}
-            download={download !== false ? download : null}
```

Comment:
Is this line needed if the attribute is being set in the watch handler as well?

Reply:
Is it needed in the componentWillLoad then? Doesn't sound like that's applying it properly?

Reply:
Removed it from the componentWillLoad and it's still working as expected